.circleci/config.yml

```diff
@@ -14,8 +14,8 @@ jobs:
       - run:
           name: Build
           command: |
-            make all -j2
-            make test -j2
+            make all -j4
+            make test -j4
```

Comment:
Parallelism for tests doesn't matter; they all run serially since the make target just executes a bash script. So I'd drop the `-j` argument here.

Reply:
`make test` actually builds the test binaries, which will benefit from more parallelism. But parallelism doesn't make for `make test-run`.